projects/gfbio-app/src/app/app.component.html

```diff
@@ -0,0 +1,53 @@
+<mat-sidenav-container fullscreen>
+    <!--    <mat-sidenav position="end" mode="side" (openedChange)="mapComponent.resize()">-->
+    <mat-sidenav position="end" mode="side">
+        <wave-sidenav-container></wave-sidenav-container>
+    </mat-sidenav>
+    <div class="time-container" fxLayoutAlign="space-around center">
+        <wave-zoom-handles (zoomIn)="map.zoomIn()" (zoomOut)="map.zoomOut()" class="mat-elevation-z4"></wave-zoom-handles>
+        <wave-small-time-interaction class="mat-elevation-z4"></wave-small-time-interaction>
+        <!--        <wave-ticker-interaction *ngIf="config.COMPONENTS.PLAYBACK.AVAILABLE" class="mat-elevation-z4"></wave-ticker-interaction>-->
```

Comment:
You added time ticker to the config but this component is commented out.
You could either remove it from the config and here or bring it back here :thinking: .

Reply:
I copied the gfbio-app from the geoengie-app and I suspect there is a reason that it is commented out.

Reply:
0b98e8eb72d8e07b75d819f61550225bbc29ad7d